pages/science/thermodynamics.rst

```diff
@@ -0,0 +1,60 @@
+.. slug: thermodynamics
+.. has_math: true
+.. title: Calculating phase and species thermodynamics
+
+.. jumbotron::
+
+   .. raw:: html
+
+      <h1 class="display-3">Calculating thermodynamic properties in Cantera</h1>
+
+   .. class:: lead
+
+      Here, we describe how Cantera uses species and phase information to calculate thermodynamic properties. Thermodynamic properties typically depend on information at both the species and phase levels.
+
+      - The user must specify a thermodynamic model for each species and provide inputs that inform how species-specific properties are calculated (e.g. as a function of temperature).
```

Comment:
> I prefer to avoid Latin abbreviations

👍🏼

> The other thing on this line is it might be nice to clarify which are species-specific properties, since you've clarified which ones are phase-average properties.

I can add some examples (reference Cp, h, and s values as a function of T should cover it).

Regarding giving a sense of how species and phase properties relate: I feel like there's a catch-22, here: anything that I can describe as being _generally_ true here would not be something that a phase model provides (if it is always true, then it can just be implemented generally for all `Phase` models).